source/install/cloud-changelog.md

```diff
@@ -4,12 +4,40 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-10-27](#release-2022-10-27)
 - [Release 2022-10-20](#release-2022-10-20)
 - [Release 2022-10-13](#release-2022-10-13)
 - [Release 2022-10-06](#release-2022-10-06)
 - [Release 2022-09-15](#release-2022-09-15)
 - [Release 2022-09-08](#release-2022-09-08)
-- [Release 2022-09-01](#release-2022-09-01)
+
+## Release 2022-10-27
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to create a new channel along with a new board associated with the created channel.
+ - Added the ability for end users to notify their Admins to reactivate a paid plan during the delinquency period by clicking “Notify admin to update billing”.
+ - Added markdown formatting for hyperlinks when pasted into the text editor.
+ - Email notifications now also include Slack attachments.
+
+#### Administration
+ - Added a new section in the **System Console** for products. For now, it only contains Boards-specific settings.
```

Comment:
Where in the System Console is the setting located in?

Reply:
It's a new section called "products" but we don't plan to enable the FF in cloud right now. So I don't believe there is a need to mention this.